_posts/2020-09-17-using-tettra.md

```diff
@@ -0,0 +1,86 @@
+
+However, sometimes a team member wants to see something in the knowledge base but is not sure what's the best way to add it. In that case, they can create a suggestion and either assign it to the best team member to write a page or to the knowledge base admin, who'll make sure the suggestion is addressed.
+
+<img src="/blog/assets/images/tettra/tettra-suggestion.jpg" alt="tettra suggestion" class="medium-img">
+
+This way we make sure all content added to the knowledge base is accurate and we also make sure we address all concerns from the team and add all the content they need there.
+
+## Adding a Category
+
+In order to make sure our knowledge base doesn't get out of order with too many categories and folders, we have a procedure in place to add new categories.
+
+Whenever a team member wants to add a new category to the knowledge base or remove a category from it, they can create a suggestion and assign it to the knowledge base admin, who'll make sure the suggestion is addressed.
+
+If a new category is created or an existing one is removed, the Knowledge Base Structure page also needs to be updated to reflect it.
+
+## Different Types of Content
+
+When talking about text, we can add content to Tettra directly through a page or a link to an external Google Doc or Sheet. However, when it comes to content in the form of videos, PDF documents, image files or audio, we need a place to store those files.
+
+We use Google Drive for that, so the content is organized and available to the whole team. However, links to those files exist inside a Tettra page so everything is accessible from within the knowledge base. This way, we're able to keep the content centralized.
+
+## Slack Integration
+
+Tettra integrates nicely with [Slack](https://slack.com), our tool of choice for internal communication. Since our team is in Slack a lot, we decided integrating our knowledge base with it was pivotal.
+
+The Slack integration allows us to notify a Slack channel every time a new page is published or an existing page is updated. We have specific categories linked to specific channels and when a page or changes to a page are published, we just need to click a button to notify the team on Slack.
+
+<img src="/blog/assets/images/tettra/slack-notify.jpg" alt="tettra notify slack" class="medium-img">
+
+Through the integration we can also easily search for content. Tettra offers a variety of Slack commands that allow us to search for content, add a new page or create a new suggestion.
+
+Finally, the integration also adds the Tettra Wiki bot, which notifies a user in Slack when they are tagged in a Tettra page or suggestion.
+
+<img src="/blog/assets/images/tettra/slack-notification.jpg" alt="tettra Slack notification" class="medium-img">
+
+## Custom Integration
+
+When we thought of the knowledge base, it was important to us to connect it to our blog posts, as it contains a great deal of information on our procedures and how we use the tools we choose to use.
```

Comment:
@ABizzinotto You don't explicitly mention GitHub, but maybe you could mention it? Our blog is an OSS project in GitHub and we had to write a custom integration for the two services to work together. Also, maybe you can mention that one of our core values is "Open by Default" and we needed to be able to index our two blogs?

Reply:
Good idea 👍